Libraries/oneMKL/binomial/src/binomial_main.cpp

```diff
@@ -61,7 +61,13 @@ void Binomial<DATA_TYPE>::check() {
       std::printf("L1 norm: %E\n", errorVal = sum_delta / sum_ref);
     else
       std::printf("Avg. diff: %E\n", errorVal = sum_delta / opt_n);
-    std::printf((errorVal < 5e-4) ? "TEST PASSED\n" : "TEST FAILED\n");
+    if(errorVal < 5e-4){
+        std::printf("TEST PASSED\n");
+    }
+    else{
+        std::printf("TEST FAILED\n");
+        exit(1);
```

Comment:
should exit also be std::exit?

Reply:
aligned with other samples, ok

Reply:
It can be, but I was going to align with other samples where we already have `exit` without `std`